tests/app/questionnaire/test_router.py

```diff
@@ -216,15 +218,40 @@ def test_return_to_final_summary_next_location_url(self):
 
         self.assertEqual(next_location, expected_location)
 
+    def test_last_block_on_path_routes_to_first_incomplete_location_in_section_when_section_not_complete(
```

Comment:
test_return_to_first_incomplete_location_when_last_block_in_section_in_progress?

Reply:
Ok, this one is as you suggested now👍